AppendBlobReader/src/appendblobreader.json

```diff
@@ -132,63 +149,80 @@
             "type": "Microsoft.Resources/deployments",
             "resourceGroup": "[parameters('StorageAccountResourceGroupName')]",
             "dependsOn": [
-                "[resourceId('Microsoft.Web/sites/', parameters('sites_blobreaderconsumer_name'))]"
+                "[variables('appendBlobProcessor_resourceId')]",
```

Comment:
do we need appendBlobProcessor_resourceId ? appendblob processor does not need storage account read permissions

Reply:
agreed, removed